src/routes/cartRouter.js

```diff
@@ -10,6 +10,10 @@ route.post("/", cartController.addToCart);
 route.delete("/:userId", cartController.removeFromCart);
 
 // Remove the Specific Item From cart
+route.patch("/items/:productId", cartController.removeSpecificItem);
+
+// Decrement the quantity from cart
+route.patch("/reduceQuantity/:productId",cartController.reduceQuantity)
```

Comment:
don't use camel case words for sub-routes, use "-" separated names like `reduce-quantity` etc.